### PR TITLE
Remove Admin Authentication settings

### PR DESCRIPTION
**Pull request by pcustic**
## Description

This PR removes Admin authentication provider settings in the System configuration page.

## Motivation and Context

We are removing Google OAuth authentication provider in the CM and with that all the external admin authentication provider code. Since that means we are also removing endpoints for admin auth edit we need to remove these options in the admin UI.

[Notion](https://www.notion.so/lyrasis/Remove-google-oauth-admin-authentication-provider-567b21054fc844cd88415ca52ba66bb2)
[CM PR](https://github.com/ThePalaceProject/circulation/pull/802)

## How Has This Been Tested?

Existing tests passed. I'll try to run the admin UI locally.

## Checklist:

- [ ] I have updated the documentation accordingly.
- [x] All new and existing tests passed.

### CODE REVIEW

**Comment by pcustic** [comment]
@jonathangreen @tdilauro @ray-lee I don't have access rights to assign people to PR so think of this as an PR review request :)

**Review by io7m — APPROVED** [review]
That looks removed to me. :+1:

File: src/stylesheets/app.scss
Comment (tdilauro) [inline src/stylesheets/app.scss:13]: It doesn't look like the corresponding `admin_auth_service_edit_form.scss` file has been removed. Do we need to keep that around for anything else?

**Review by tdilauro — CHANGES_REQUESTED** [review]
Looks pretty good. Just one small issue to resolve.

File: src/stylesheets/app.scss
Comment (pcustic) [reply src/stylesheets/app.scss:13]: Good catch 👍 https://github.com/ThePalaceProject/circulation-admin/pull/75/commits/57649772776b7fea9d2c9358550fe76f90312706

**Comment by pcustic** [comment]
@tdilauro I can't merge this so if you feel this is good to go can you please merge it?
Also, the original [PR in the CM](https://github.com/ThePalaceProject/circulation/pull/802) is not reviewed/approved yet so I'm not sure if that should be reviewed first?

**Review by tdilauro — APPROVED** [review]

**Pull request merged**
